lib/vsc/utils/generaloption.py

```diff
@@ -583,9 +595,53 @@ def _is_enable_disable(x):
             for opt in self._get_all_options():
                 # remove all long_opts with ENABLE/DISABLE naming
                 opt._long_opts = [x for x in opt._long_opts if not _is_enable_disable(x)]
+        return fh
 
+    def print_help(self, fh=None):
+        """Intercept print to file to print to string and remove the ENABLE/DISABLE options from help"""
+        fh = self.check_help(fh)
         OptionParser.print_help(self, fh)
 
+    def print_rsthelp(self, fh=None):
+        """ Print help in rst format """
+        fh = self.check_help(fh)
+        result = []
+        if self.usage:
+            result.append("Usage: ``%s``" % self.get_usage().replace("Usage: ", '').strip())
+            result.append('')
+        if self.description:
+            result.append(self.description)
+            result.append('')
+
+        result.append(self.format_option_rsthelp())
+
+        rsthelptxt = '\n'.join(result)
+        if fh is None:
+            fh = sys.stdout
+        fh.write(rsthelptxt)
+
+    def format_option_rsthelp(self, formatter=None):
+        """ Formatting for help in rst format """
+        if not formatter:
+            formatter = self.formatter
+        formatter.store_option_strings(self)
+
+        res = []
+        titles = ["Option", "Help"]
+
+        all_opts = [("Options", self.option_list)] + [(group.title, group.option_list) for group in self.option_groups]
```

Comment:
self.option_list only contains `--help` and co + `--version`, maybe rename `"Options"` to `"Help options"` ?